lib/caqti_request.mli

```diff
@@ -198,18 +197,18 @@ val pp_with_param :
     following can be used to existentially pack the parameter types along with
     the corresponding parameter values to allow collecing them incrementally:
     {[
-module Dynparam = struct
-  type t = Pack : 'a Caqti_type.t * 'a -> t
-  let empty = Pack (Caqti_type.unit, ())
-  let add t x (Pack (t', x')) = Pack (Caqti_type.tup2 t' t, (x', x))
-end
+      module Dynparam = struct
+        type t = Pack : 'a Caqti_type.t * 'a -> t
+        let empty = Pack (Caqti_type.unit, ())
+        let add t x (Pack (t', x')) = Pack (Caqti_type.tup2 t' t, (x', x))
+      end
```

Comment:
IIRC the extra indentation showed up in the generated docs in old times, but it looks good now and definitely better to look at in the source code.